test/manual/kerberos.test.js

```diff
@@ -0,0 +1,352 @@
+'use strict';
+
+const format = require('util').format;
+const test = require('../functional/shared').assert;
+const { MongoClient } = require('../../src');
+
+// You need to set up the kinit tab first
```

Comment:
Let's remove all of this, we know where to look and we don't want to give any details externally